e2e/process-services-cloud/start-process-cloud.e2e.ts

```diff
@@ -109,22 +108,11 @@ describe('Start Process', () => {
         startProcessPage.enterProcessName(processName);
         expect(startProcessPage.checkStartProcessButtonIsEnabled()).toBe(true);
         startProcessPage.clickStartProcessButton();
-        processCloudDemoPage.clickOnProcessFilters();
 
         processCloudDemoPage.runningProcessesFilter().clickProcessFilter();
         expect(processCloudDemoPage.getActiveFilterName()).toBe('Running Processes');
         processCloudDemoPage.processListCloudComponent().checkContentIsDisplayedByName(processName);
 
     });
 
-    it('[C309875] Should display the processId when Process Definition has process name missing', async () => {
```

Comment:
Why is this test deleted?

Reply:
is not valid anymore

Reply:
also in testrail has been already deleted

Reply:
Perfect! Thanks